src/Lightest.Api/Controllers/UploadsController.cs

```diff
@@ -7,6 +7,8 @@
 using Lightest.Data;
```

Comment:
Is this the only file which uses upload code? Use "Find All References" to  verify.